recipes/climix/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "climix" %}
+{% set version = "0.18.0rc2" %}
```

Comment:
It looks like this package has normal releases. Given that, we really don't want to be packaging release candidates on the main channel.

Otherwise, the recipe looks fine.

Reply:
Well, kind of. I am the main author of the package and the thing that is holding back the next proper release is the documentation saying "to install, do 'mamba install climix'".

Chicken and egg, but if you prefer to invert the relationship, we can finish the docs as is and make them true retroactively.

Reply:
That would be preferred, yes. The recipe is ready, so it appearing on conda-forge should be very quick.

Reply:
Done.